Adds Windows support (#235)
src-d/engine: README.md

  sudo pacman -S docker
  ```

> 

Reviewer: ```suggestion
```

- [Docker for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows). Make sure to read the system requirements [here](https://docs.docker.com/docker-for-windows/install/).

#### 2. Install source{d} Engine

> Download the **[latest release](https://github.com/src-d/engine/releases)** for MacOS (Darwin) or Linux.

Reviewer: ```suggestion
Download the **[latest release](https://github.com/src-d/engine/releases)** for MacOS (Darwin), Linux or Windows.
```

**Windows:**

Extract the tar file with the tool you prefer.

> Open your powershell and move the `srcd.exe` file it into any path listed in `PATH`.

Reviewer: ```suggestion
Copy  `srcd.exe` into a directory pointed by your `%path%` environment variable to make it available from anywhere.
```

> Open your powershell and move the `srcd.exe` file it into any path listed in `PATH`.

Reviewer 2: Can we simplify this section? Is there a default path we can use here to make it ready to copy&paste?

Author: Here's the content of `PATH` on the dev machine:
```powershell
PS C:\Users\Windows10\go> $env:path.split(";")
C:\ProgramData\DockerDesktop\version-bin
C:\Program Files\Docker\Docker\Resources\bin
C:\Windows\system32
C:\Windows
C:\Windows\System32\Wbem
C:\Windows\System32\WindowsPowerShell\v1.0\
C:\Windows\System32\OpenSSH\
C:\Program Files\Git\cmd
C:\Go\bin
C:\ProgramData\chocolatey\bin
C:\Users\Windows10\AppData\Local\Microsoft\WindowsApps
C:\Users\Windows10\go\bin
```

I think that `C:\Windows\system32` could be a nice candidate.

Reviewer: When I was young, most of apps suggested you add your directory into your path env... It could be an alternative...
```powershell
SET PATH=%PATH%;{DIR_WHERE_SRCD.EXE_WAS_UNPACKED};
```

Author: @dpordomingo yup that is an alternative. I avoided explaining that just to make the quickstart easier.

Reviewer: manually copying a downloaded thing into `system32`? wow... it might be scary...

Author: > it might be scary...
😂 
I actually don't know windows, but I guess it doesn't differ too much from copying a downloaded thing into `/usr/local/bin` in unix systems. But maybe I'm wrong.

Reviewer: [Go installation guide](https://golang.org/doc/install/source#install) in Windows suggests you to add go bin directory in your `PATH`, so it seems a default behavior for windows developers...

Reviewer: I think that `C:\Windows\system32` is not a user directory. I wonder if even the antivirus could complain about it...

Author: Ok. We just have to decide where to put it then.

Author: > I wonder if even the antivirus could complain about it...

Good point! 👍 
I haven't thought about that, it's been years that I don't run an antivirus...

Author: What about `C:\Program Files\Engine` (maybe too generic)? or `C:\Program Files\srcd`?

Reviewer: If there is no permissions problem, both lgtm

Reviewer 2: I prefer `C:\Program Files\srcd`, it sounds more generic, and it could be used for other tools we may release in the future.

> Move it into your local bin folder to be executable from anywhere:

Reviewer: dupe
```suggestion
```

```powershell
mv engine_windows_amd64\srcd.exe some\path
```

> You can list the paths by running the following command:

Reviewer: ```suggestion
You can list the directories pointed by your `%path%` environment variable by running the following command:
```
